sources/packages/backend/apps/api/src/route-controllers/education-program-offering/_tests_/e2e/education-program-offering.institutions.validateOffering.e2e-spec.ts

```diff
@@ -0,0 +1,125 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import {
+  Institution,
+  InstitutionLocation,
+  OfferingIntensity,
+  OfferingTypes,
+  User,
+} from "@sims/sims-db";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  createFakeInstitutionLocation,
+  OfferingDeliveryOptions,
+} from "@sims/test-utils";
+import {
+  createTestingAppModule,
+  BEARER_AUTH_TYPE,
+  InstitutionTokenTypes,
+  getInstitutionToken,
+  authorizeUserTokenForLocation,
+  getAuthRelatedEntities,
+  createFakeEducationProgram,
+} from "../../../../testHelpers";
+import * as request from "supertest";
+
+describe("EducationProgramOfferingInstitutionsController(e2e)-validateOffering", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+  let collegeF: Institution;
+  let collegeFLocation: InstitutionLocation;
+  let collegeFUser: User;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+    const { institution, user: institutionUser } = await getAuthRelatedEntities(
+      db.dataSource,
+      InstitutionTokenTypes.CollegeFUser,
+    );
+    collegeF = institution;
+    collegeFUser = institutionUser;
+    collegeFLocation = createFakeInstitutionLocation(collegeF);
+    await authorizeUserTokenForLocation(
+      db.dataSource,
+      InstitutionTokenTypes.CollegeFUser,
+      collegeFLocation,
+    );
+  });
+
+  it("Should approve an offering when passed valid data.", async () => {
```

Comment:
Should auto approve ?

Reply:
I meant the validation should be "approved". I'm going to change the text.